adapt/expander.py

```diff
@@ -156,24 +350,64 @@ def end_token_index():
         for i in xrange(len(tags)):
             tag = tags[i]
 
-            if len(overlapping_spans) > 0 and end_token_index() >= tag.get('start_token'):
+            if len(overlapping_spans) > 0 and end_token_index(
```

Comment:
what's with the weird line breaks here? Can you fix this? I don't think most of these types of changes are necessary for pep8.